### PR TITLE
Use canMidAirMorph Throughout

### PR DESCRIPTION
**Pull request by kjbranch**

### CODE REVIEW

File: region/maridia/outer/Main Street.json [hunk 1]
```diff
@@ -2446,7 +2445,8 @@
       "name": "Suitless Bootless Mid-Air Morph",
       "requires": [
         "canSuitlessMaridia",
-        "canDisableEquipment"
+        "canDisableEquipment",
+        "canMidAirMorph"
```
Comment (osse101) [inline region/maridia/outer/Main Street.json:2449]: [7, 5] too

File: region/wreckedship/main/Wrecked Ship East Super Room.json [hunk 2]
```diff
@@ -219,17 +219,20 @@
       "link": [1, 2],
       "name": "Base",
       "requires": [
-        "Morph",
         "h_canDestroyBombWalls",
-        "f_DefeatedPhantoon"
+        "f_DefeatedPhantoon",
+        {"or": [
+          "canTrivialMidAirMorph",
```
Comment (osse101) [inline region/wreckedship/main/Wrecked Ship East Super Room.json:225]: One could roll off of the robot

File: region/wreckedship/main/Wrecked Ship East Super Room.json [hunk 3]
```diff
@@ -260,7 +268,7 @@
       "link": [1, 2],
       "name": "WS East Supers Robot Clip (Left to Right)",
       "requires": [
-        "Morph",
+        "canTrivialMidAirMorph",
```
Comment (osse101) [inline region/wreckedship/main/Wrecked Ship East Super Room.json:271]: This makes it seem like canKago should require canMidairMorph

File: region/crateria/east/West Ocean.json [hunk 4]
```diff
@@ -1747,15 +1747,25 @@
       "link": [10, 13],
       "name": "Base",
       "requires": [
-        {"obstaclesCleared": ["B"]}
+        {"or": [
+          "h_canBombThings",
+          {"obstaclesCleared": ["B"]}
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/crateria/east/West Ocean.json:1752]: Yeah, its a strat for coming up through the crumble blocks and coming back down. Morph is required. Going up instead has canRiskPermanentLossOfAccess

**Pull request merged**